addons/account/wizard/account_payment_register.py

```diff
@@ -875,6 +915,11 @@ def _create_payments(self):
         return payments
 
     def action_create_payments(self):
+        self.ensure_one()
+        # Check that we do not make a payment to a payment method that requires an account if the account isn't trusted.
+        if self.total_payments_amount == 1 and self.require_partner_bank_account and not self.partner_bank_id.allow_out_payment:
```

Comment:
so if I have 2 payments, there is no verification ?

Reply:
If you are processing multiple payments, a warning is shown on the screen. Payments that are not ok are ignored while the others are created.